packages/scandipwa/src/store/Wishlist/Wishlist.dispatcher.js

```diff
@@ -77,7 +77,9 @@ export class WishlistDispatcher {
                             qty: quantity
                         } = wishlistItem;
 
-                        const priceRange = getPriceRange(product, price, price_without_tax);
+                        const { discount } = product.price_range.minimum_price;
```

Comment:
Please don't have such long chains.
It's better to have it like this:
```javascript
const {
  price_range: {
    minimum_price: {
      discount = 0
    } = {}
  } = {}
} = product;
```
This will allow getting easier other data from a product. Also, specify the fallbacks if the product isn't really a product we expect.